### PR TITLE
aio: fix slow docs-test run

### PR DESCRIPTION
**Pull request by petebacondarwin**
Parsing all the node_modules in the example boilerplate was causing doc gen to run slowly,
which is causing flaky integration tests.

Also not all the tests were being run as the Jasmine CLI config was not correct.

### CODE REVIEW

File: aio/tools/transforms/test.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const Jasmine = require('jasmine');
+const jasmine = new Jasmine({ projectBaseDir: __dirname });
+jasmine.loadConfig({ spec_files: ['**/*.spec.js'] });
+jasmine.execute();
```
Comment (gkalpak) [inline aio/tools/transforms/test.js]: Why is it not possible to run the tests via the Jasmine CLI?
Adding a comment (explaining that) in this file would be helpful for future reference.

Comment (petebacondarwin) [inline aio/tools/transforms/test.js]: Will do

Comment (petebacondarwin) [inline aio/tools/transforms/test.js]: Done

Comment (gkalpak) [inline aio/tools/transforms/test.js]: Interestingly, the glob works correctly with the Jasmine CLI on my Windows, but I can see it doesn't on Travis 😕
Happy to go with whatever works :smiley:

💯 on finding that out. I never noticed not all tests were run on Travis

**Review by gkalpak — APPROVED** [review]

**Pull request merged**